UnoCheck/Solutions/LinuxOtherDistGitCliSolution.cs

```diff
@@ -0,0 +1,26 @@
+﻿using DotNetCheck.Models;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class LinuxOtherDistGitCliSolution : Solution
+	{
+		public override async Task Implement(SharedState sharedState, CancellationToken cancellationToken)
+		{
+			await base.Implement(sharedState, cancellationToken);
+
+			var ps = new ProcessStartInfo(PythonDownloadUrl)
+			{
+				UseShellExecute = true,
+				Verb = "open"
+			};
+			_ = Process.Start(ps);
```

Comment:
Can you validate that this works on a Linux environment? I'm not sure if the verb is supported.

Reply:
It needed to fix on Fedora. Done!